whois-api/src/main/java/net/ripe/db/whois/api/fulltextsearch/ElasticFulltextSearch.java

```diff
@@ -150,6 +159,14 @@ protected SearchResponse doSearch() throws IOException {
         }.search();
     }
 
+    private UserSession getUserSession(final String ssoToken) {
```

Comment:
This code is repeated, can you move it to the SsoTokenTranslator class? e.g. call it `translateSsoTokenOrNull(...)`